PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,4 +10,4 @@ references to all relevant github issues._)
 ### Result
 
 (_describe how your changes affect the end-user behavior of the system. this section is
-optional, and should generally be summarized in the title of the pull request._)
\ No newline at end of file
+optional, and should generally be summarized in the title of the pull request._)
```

Comment:
I opened this file earlier today, and now my IDE won't stop adding a new line! Even after restarting it. It's not wrong to have a new line, so I included it here for my sanity :)